x-pack/plugins/canvas/common/lib/index.js

```diff
@@ -26,3 +26,28 @@ export * from './pivot_object_array';
 export * from './resolve_dataurl';
 export * from './unquote_string';
 export * from './url';
+
+// Eslint complains about these being undefined with doing export *
+// Probably related to constants moving to typescript?
+export {
```

Comment:
This is a bit strange.  

This is a JS file that's doing an export * from 'some-ts-file'

ESLint errors when trying to access one of the exports from that file via this file with a `name does not exist` type error.  

Everything works correctly, it's just that eslint is throwing an error. 

Explicitly exporting like this fixes things, but we lose the flexibility of exporting everything.

Reply:
Pro tip: gotta rename this as a TS file for the types to pass through.  Right now, you're importing a typed collection, but exporting an untyped collection since the file is a JS file.